addons/point_of_sale/i18n/point_of_sale.pot

```diff
@@ -7079,6 +7079,14 @@ msgid ""
 "the form view in studio if no PoS order exist"
 msgstr ""
 
+#. module: point_of_sale
+#. odoo-python
+#: code:addons/point_of_sale/models/stock_picking.py:0
+#, python-format
+msgid ""
+"You cannot archive this Operation Type as it is used by a POS configuration."
+msgstr ""
+
```

Comment:
This should be automatically generated. Try checking out: https://www.odoo.com/documentation/16.0/developer/howtos/translations.html

Reply:
If I am not wrong you are referring to this https://www.odoo.com/documentation/16.0/developer/howtos/translations.html#implicit-exports

But I don't think the string inside the `ValidationError` will be implicitely Exported.

What do you say?

Reply:
Sorry to be vague. But I'm actually referring to this: https://www.odoo.com/documentation/16.0/developer/howtos/translations.html#exporting-translatable-term

Instead of you manually writing the translation diff, use the exported file and replace the existing.

Reply:
Hello @caburj ,
Required changes are done, can you please review them?